src/tlv/TlvObject.ts

```diff
@@ -97,8 +107,8 @@ export const TlvObject = <F extends TlvFields>(fields: F) => new ObjectSchema(fi
 /** List TLV schema. */
 export const TlvList = <F extends TlvFields>(fields: F) => new ObjectSchema(fields, TlvType.List);
 
-/** Object TLV mandatory field. */
-export const TlvField = <T>(id: number, schema: TlvSchema<T>) => ({ id, schema, optional: false }) as FieldType<T>;
+/** Object TLV mandatory field. A fallback value can be defined for backward compatibility.  */
```

Comment:
This would be my (non native speaking) try :-)

```suggestion
/** 
  * Object TLV mandatory field. Optionally provide a fallback value to initialize the field value when devices omit
  * providing a value against the specifications or in special usecases. Make sure to use a value that is an equivalent
  * to the value being empty.
  */
```